osu.Game/Online/API/Requests/GetUserMessagesRequest.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System.Collections.Generic;
+using osu.Framework.IO.Network;
+using osu.Game.Online.Chat;
+
+namespace osu.Game.Online.API.Requests
+{
+    public class GetUserMessagesRequest : APIRequest<List<Message>>

```

Comment:
`GetMessagesPrivateRequest`

Reply:
I'm pretty sure most of this code is shared with `GetMessagesRequest`, so make a common abstract base class?